CHANGELOG.md

```diff
@@ -1,3 +1,259 @@
+
+#### Platform updates
+
+##### AKS
+
+- Kubernetes version has been updated to `1.18.8`
+  ([#1071](https://github.com/kinvolk/lokomotive/pull/1071)).
+
+##### Baremetal
+
+- Expose CNI MTU on the baremetal platform
+  ([#977](https://github.com/kinvolk/lokomotive/pull/977)).
+
+#### New components
+
+- Component `web-ui`
+  ([#981](https://github.com/kinvolk/lokomotive/pull/981)),
+  ([#1100](https://github.com/kinvolk/lokomotive/pull/1100))
+  from [headlamp](https://github.com/kinvolk/headlamp).
+- Component `inspektor-gadget`
+  ([#1076](https://github.com/kinvolk/lokomotive/pull/1076))
+  from [inspektor-gadget](https://github.com/kinvolk/inspektor-gadget/).
+
+#### Component updates
+
+- Update Velero component for Packet (OpenEBS and restic plugin support)
+  ([#881](https://github.com/kinvolk/lokomotive/pull/881)).
+- istio-operator: Update to 1.7.3
+  ([#1086](https://github.com/kinvolk/lokomotive/pull/1086)).
+- prometheus-operator: Update grafana, kube-state-metrics and node_exporter
+  ([#963](https://github.com/kinvolk/lokomotive/pull/963)).
+- cert-manager: Update to 1.0.3
+  ([#1114](https://github.com/kinvolk/lokomotive/pull/1114)).
+
+#### Terraform updates
+
+- Upgrade to Terraform 0.13
+  ([#824](https://github.com/kinvolk/lokomotive/pull/824)).
+
+#### Features
+
+- Encryption support to in-cluster pod traffic
+  ([#911](https://github.com/kinvolk/lokomotive/pull/911)).
+- AWS, Packet, Baremetal: use Docker instead of rkt for host containers
+  ([#946](https://github.com/kinvolk/lokomotive/pull/946)).
+- Change labels and taints format from string to structured
+  ([#1042](https://github.com/kinvolk/lokomotive/pull/1042)).
+- prometheus-operator: Add external_url
+  ([#964](https://github.com/kinvolk/lokomotive/pull/964)).
+
+#### Docs
+
+- Concept document for admission webhook
+  ([#943](https://github.com/kinvolk/lokomotive/pull/943)).
+- Coding style guide
+  ([#953](https://github.com/kinvolk/lokomotive/pull/953)).
+- Metallb: Clarify address_pools knob
+  ([#996](https://github.com/kinvolk/lokomotive/pull/996)).
+- How to guide on backing up and restoring rook-ceph volumes with Velero
+  ([#1048](https://github.com/kinvolk/lokomotive/pull/1048)).
+
+#### Bug fixes
+
+- bootkube: feed output using local rather than local_file content
+  ([#1021](https://github.com/kinvolk/lokomotive/pull/1021)).
+- Dex: fix pod reload on config change
+  ([#1040](https://github.com/kinvolk/lokomotive/pull/1040)).
+- Metallb: Add missing autodiscovery labels
+  ([#990](https://github.com/kinvolk/lokomotive/pull/990)).
+- Gangway: add a ServiceAccount
+  ([#1104](https://github.com/kinvolk/lokomotive/pull/1104)).
+- If there is more than one component installed in single namespace, `lokoctl` will now refuse to remove then namespace while running `lokoctl component --delete` with `--delete-namespace` flag ([#1093](https://github.com/kinvolk/lokomotive/pull/1093)).
+
+#### Development
```

Comment:
Personally I'm not sure it's useful to include information about implementation changes in the release notes unless they significantly affect the users (e.g. a noticeable change in performance, much better reliability etc.).

As a user who reads the release notes, I am interested mainly in the following:

- Things I will **get** by upgrading.
- Things I have to **worry about** when upgrading.

As a user I don't care too much about things I don't see when using the product (e.g. refactoring).

Additionally, my interpretation of "development-related changes" is "changes which are relevant to Lokomotive developers". Looking at past releases it seems that this was what we initially meant, but then we shifted to mentioning small under the hood fixes.

We may want to reconsider this section and maybe include only things like changes to testing, CI, development processes etc.

Reply:
I agree with @johananl here. But I think we can consider that doing next release to avoid blocking this one further.